### PR TITLE
Docs(web-twig): Remove misplaced quotes from Toast Collapsible docs #DS-1420

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Toast/README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ set the `isCollapsible` prop to `false`.
 👉 Please note that the initial scroll position is always at the **top** of the queue.
 
 ```html
```
Comment (literat) [inline packages/web-twig/src/Resources/components/Toast/README.md:121]: This is a Prettier error, e.g. it parses this example as HTML (note the ` ```html `). Change also this to `jsx` or `tsx` and it should work correctly.

Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Toast/README.md:121]: aha! tru!

Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Toast/README.md]: I added another commit where I transfer all code snippets in Twig to use twig syntax

Comment (literat) [reply packages/web-twig/src/Resources/components/Toast/README.md]: 👏 Great, thanks 👍

**Review by literat — APPROVED** [review]

**Comment by literat** [comment]
I have sent a formatting fixup.

**Review by curdaj — APPROVED** [review]

**Pull request merged**